SwinjectTests/AssemblerSpec.swift

```diff
@@ -0,0 +1,252 @@
+//
+//  AssemblerSpec.swift
+//  Swinject
+//
+//  Created by mike.owens on 12/9/15.
+//  Copyright © 2015 Swinject Contributors. All rights reserved.
+//
+
+import Foundation
+import Quick
+import Nimble
+import Swinject
+
+class AssemblerSpec: QuickSpec {
```

Comment:
Type Body Length Violation: Type body should be span 200 lines or less: currently spans 237 lines (type_body_length)